tasks.py

```diff
@@ -0,0 +1,18 @@
+from invoke import task
+
+@task
+def regenerate_models(c):
```

Comment:
This is just a convenience script for generating the pydantic models from the schemas submodule. This functionality could be pushed into the server code (and then `tasks.py` can just import it) and run in e.g., GH actions or server startup automatically.

Ideally it would also generate the `models/__init__.py` such that the version is set to match the commit hash of the `schemas` submodule.

For now, its probably better to keep this process manual (i.e., you need to call `invoke regenerate-models` yourself), I think, so that we can make sure the example data doesn't go stale.